docs/networkpolicy-recommendation.md

```diff
@@ -63,17 +63,17 @@ To see all options and usage examples of these commands, you may run
 The `theia policy-recommendation run` command triggers a new policy
 recommendation job.
 If a new policy recommendation job is created successfully, the
-`recommendation ID` of this job will be returned:
+`name` of this job will be returned:
 
 ```bash
 $ theia policy-recommendation run
-Successfully created policy recommendation job with ID e998433e-accb-4888-9fc8-06563f073e86
+Successfully created policy recommendation job with name pr-e998433e-accb-4888-9fc8-06563f073e86
 ```
 
-`recommendation ID` is a universally unique identifier ([UUID](
+The name of policy recommendation job contains universally unique identifier ([UUID](
```

Comment:
```suggestion
The name of the policy recommendation job contains a universally unique identifier ([UUID](
```

Reply:
Thanks Yongming for reviewing. Fixed. Thanks.